sqlmesh/core/config/connection.py

```diff
@@ -333,6 +333,7 @@ class SnowflakeConnectionConfig(ConnectionConfig):
         private_key_passphrase: The optional passphrase to use to decrypt `private_key` or `private_key_path`. Keys can be created without encryption so only provide this if needed.
         register_comments: Whether or not to register model comments with the SQL engine.
         pre_ping: Whether or not to pre-ping the connection before starting a new transaction to ensure it is still alive.
+        session_parameters: The optional session parameters to set for the connection.
```

Comment:
I suggest we call this `session_properties`, since this is a terminology that we already use in the model definition.

Reply:
Ah, this should match the name expected by the snowflake connector. Disregard my comment.